first draft on guide to authenticate (#4252)
meshery/meshery: docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md

---
layout: default
title: Authenticating with Meshery via CLI 
permalink: guides/mesheryctl/authenticate-with-meshery-via-cli
language: en
type: Guides
category: mesheryctl
---

> To authenticate with Meshery via mesh you have to use the command `mesheryctl system login`. On execution of this command you get a an option to choose a provider, you would have to choose `Meshery` as your provider to authenticate yourself. After doing so, a new window opens up with ways to login. 

Reviewer: ```suggestion
To authenticate with Meshery through `mesheryctl` you will use the command `mesheryctl system login`. Upon execution of this command, select your Provider of choice, then authenticate to your chosen Provider.
```

Reviewer: It'd be good to include a tip / note "What is a Meshery Provider?" using `{% include alert.html %}`

## Get your Token

> ##### There are 2 ways to get your token:

Reviewer: ```suggestion
You can retrieve your authentication token from either of Meshery's two clients: the CLI or the UI. 
```

> - Get your token through **Meshery Dashboard**, from the `Get Token` option.

Reviewer: ```suggestion
- Get your token through [Meshery UI](/concepts/architecture/ui), from the `Get Token` option.
```
